src/soot/jimple/toolkits/callgraph/InvokeCallSite.java

```diff
@@ -0,0 +1,54 @@
+package soot.jimple.toolkits.callgraph;
+
+import soot.Local;
+import soot.SootMethod;
+import soot.jimple.InstanceInvokeExpr;
+import soot.jimple.Stmt;
+import soot.jimple.toolkits.callgraph.ConstantArrayAnalysis.ArrayTypes;
+
+public class InvokeCallSite {
+	public static final int MUST_BE_NULL = 0;
```

Comment:
These constants overlap with the NullnessAnalysis constants but they are protected so they are duplicated here.